### ISSUE TITLE
It would be really cool....

### ISSUE THREAD

**Issue by kingram6865** [roldanjr/productivity-timer#109]
**Is your feature request related to a problem? Please describe.**
No. Pure featured request.
I would love to be able to save my completed tasks to a database.

**Describe the solution you'd like**
It would be good in tracking workflows over time. There doesn't seem to otherwise
be a way to save task start-end, and show work-rest cycles as a history.

MySQL, SQLite, Postgres, etc.

**Describe alternatives you've considered**
There aren't any alternatives other than saving to a flat file (CSV)
That may be ab easy first start.

**Additional context**
If you would like some assistance on this piece i would be happy to
contribute my time to building a database API, first for storing, then
perhaps for retrieving?

This is a great app. Found it and used it for the first time today. Was very handy.
Then I noticed I wanted to save today's session and there was no way to do that.

**Comment by roldanjr** [comment]
@kingram6865 yes, It's really cool to have that kind of feature. 
I was planning to do that but unfortunately I don't have much time for now.

Any help would be pleasing to me. And I thank you in advance @kingram6865 for that.

**Comment by kingram6865** [comment]
I'll be occupied in a coding bootcamp until November 1, so I will start working on an API for database connection in my spare moments.

When it's a bit workable, I'll send you the info. Will start the project here on github.

**Comment by roldanjr** [comment]
Yeah I understand. We're the same, I am maintaining this app using my spare time only too.

I am planning to convert this project as monorepo too and rename it to another non-generic name.

I really appreciate it @kingram6865 . Thank you so much for the effort. I salute you man.

**Comment by roldanjr** [comment]
Hello @kingram6865 , 

I just want to give you some update regarding with this project.  
I've restructured the app architecture to be more future proof.

I need to change it to a monorepo using [Lerna](https://lerna.js.org/) and [Yarn Workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) so that there's a clear separation of concerns between renderer and main process and also makes it easier for code sharing between those processes.

And it really helps reduce the bundle size of the app as well as the installers of different operating system itself.

Please check it out as soon as you can. Thank you friend.

**Comment by kingram6865** [comment]
Great. Will let you know once I have something interesting.
